src/views/Settings.vue

```diff
@@ -2,100 +2,60 @@
   <!-- content page: 设置 -->
   <div class="page" ng-init="lastfm.updateStatus()">
     <div class="site-wrapper-innerd">
-      <div class="cover-container leading-normal">
-        <SettingTitle :text="t('_LANGUAGE')" />
-        <div class="settings-content mx-7 mb-7 leading-8">
-          <SettingButton text="简体中文" @click="setLocale('zh-CN')" />
-          <SettingButton text="繁体中文" @click="setLocale('zh-TC')" />
-          <SettingButton text="English" @click="setLocale('en-US')" />
-          <SettingButton text="French" @click="setLocale('fr-FR')" />
-        </div>
-        <SettingTitle :text="t('_BACKUP_PLAYLIST')" />
-        <div class="settings-content mx-7 mb-7 leading-8">
+      <div class="cover-container leading-normal flex content-left flex-col flex-wrap gap-4 2xl:h-[calc(100vh-10rem)]">
```

Comment:
```suggestion
      <div class="cover-container leading-normal flex content-left flex-col flex-wrap gap-4 2xl:h-[calc(100vh-10rem)] 2xl:overflow-y-hidden">
```
I would recommend disable y scroll in 2xl.

Reply:
Didn't figure out any obvious effect. Can you explain more?